pkg/network/tracer/connection/tcp_close_consumer.go

```diff
@@ -129,8 +129,8 @@ func (c *tcpCloseConsumer) Start(callback func([]network.ConnectionStats)) {
 				if !ok {
 					return
 				}
-				closerConsumerTelemetry.perfLost.Add(float64(lc * netebpf.BatchSize))
-				lostCount += lc * netebpf.BatchSize
+				closerConsumerTelemetry.perfLost.Add(float64(lc))
```

Comment:
one lost event could be a full batch of connections. Perhaps this was meant to be `lc * numOfConnsInBatch`?

Reply:
Gotcha. I'll leave it up to you to decide what you think is best for your team.